application/src/static/stylesheets/fonts-ie8.css

```diff
@@ -3,6 +3,9 @@
 /*! Copyright (c) 2011 by Margaret Calvert & Henrik Kubel. All rights reserved. The font has been customised for exclusive use on gov.uk. This cut is not commercially available. */
 
 
+
```

Comment:
Is there a reson for the additional blank lines here?

Reply:
No, I have no idea how that happened...